src/lib/voiceSearchHelper/__tests__/index-test.ts

```diff
@@ -85,7 +85,7 @@ describe('VoiceSearchHelper', () => {
     voiceSearchHelper.toggleListening();
     expect(onStateChange).toHaveBeenCalledTimes(1);
     expect(voiceSearchHelper.getState().status).toEqual('askingPermission');
-    listeners.start();
+    listeners.start(); // This way, we're simulating `start` event of the SpeechRecognition.
```

Comment:
I didn't mention it into my original comment, but it worth adding it on the others listeners calls.

Reply:
https://github.com/algolia/instantsearch.js/pull/3779/commits/af65fcd0e9c1ce8122a3e1f39a3aaaa2ef68c97a